utils/node_local_test.py

```diff
@@ -1632,6 +1645,107 @@ def test_dfuse_dio_off(self):
         if dfuse.stop():
             self.fatal_errors = True
 
+    def test_daos_fs_tool(self):
+        """Create a UNS entry point"""
+
+        pool = self.pool
+        container = self.container
+        server = self.server
+        conf = self.conf
+
+        # Start dfuse on the container.
+        dfuse = DFuse(server, conf, pool=pool, container=container)
+        dfuse.start('uns-0')
```

Comment:
You should use the @needs_dfuse decorator here.

Reply:
i actually copied most of this initial part from:
test_uns_basic
which doesn't have this..

Reply:
test_uns_basic is older code moved to the new method interface, but what's special about that is that it starts and stops dfuse several times, so it isn't able to use the decorator unless/until it's split into smaller units.  Even then I think it verifies that you can access the same file various paths/settings so probably does actually need to be able to re-start dfuse.

I'm extending this test code in https://github.com/daos-stack/daos/pull/5752 to run everything twice, once with caching and once without, so if this was using the decorator it would benefit from that.  If you want to land this and I'll update this method in my PR then that's fine with me.

Reply:
i can add it. it seems my PR didn't go far in testing anyway bec of yesterday's CI/wolf issues.